adr/0001-v1alpha-schema.md

```diff
@@ -0,0 +1,57 @@
+---
+creation_date: 2021-11-24
+decision_date: 2021-11-25
+status: accepted
```

Comment:
Should this be:
```suggestion
status: superceded
```

As we've now got ADR-0003?

Reply:
superseded makes sense to me